test/test.py

```diff
@@ -7,6 +7,8 @@
 import hashlib
 import time
 
+perform_slow_tests = False
```

Comment:
should this stay in? Or maybe default to true?

Locally you can run `pytest test/test.py:: test_working_download_broken_checksum` to only run a specific test.

Reply:
Good point, I simply removed the time based test, because it might sometimes take very long...